### ISSUE TITLE
[Bug]: OPcache warning even if parameters are correct

### ISSUE THREAD

**Issue by Jolopu** [nextcloud/server#34291]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] Nextcloud Server **is** running on 64bit capable CPU, PHP and OS.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

Since NC 23 there is an OPcache warning displayed in the settings saying that opcache.max_accelerated_files, opcache.memory_consumption and opcache.interned_strings_buffer are too low. So I went ahead and doubled the values in the corresponding php.ini. The warning didn't go away, instead it's displayed with the new doubled values. No matter which values I set the warning is always displayed. I'm on NC 24.0.5 and PHP 8.1.7 running on a shared host.

![OPcache](https://user-images.githubusercontent.com/34797349/192538146-a76509cd-d1f5-407e-81a4-651bf15721c9.png)

### Steps to reproduce

1. Install NC 24.0.5 and PHP 8.1.7 on a shared host
2. Check the settings overview

### Expected behavior

No OPcache warnings

### Installation method

Community Web installer on a VPS or web space

### Operating system

Debian/Ubuntu

### PHP engine version

PHP 8.1

### Web server

Apache (supported)

### Database engine version

MariaDB

### Is this bug present after an update or on a fresh install?

Updated to a major version (ex. 22.2.3 to 23.0.1)

### Are you using the Nextcloud Server Encryption module?

Encryption is Enabled

### What user-backends are you using?

- [X] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

```shell
{    "system": {        "instanceid": "***REMOVED SENSITIVE VALUE***",        "passwordsalt": "***REMOVED SENSITIVE VALUE***",        "secret": "***REMOVED SENSITIVE VALUE***",        "trusted_domains": [            "cloud.geistigewelt.tv"        ],        "datadirectory": "***REMOVED SENSITIVE VALUE***",        "dbtype": "mysql",        "version": "24.0.5.1",        "overwrite.cli.url": "https:\/\/cloud.geistigewelt.tv",        "dbname": "***REMOVED SENSITIVE VALUE***",        "dbhost": "***REMOVED SENSITIVE VALUE***",        "dbport": "",        "dbtableprefix": "oc_",        "mysql.utf8mb4": true,        "dbuser": "***REMOVED SENSITIVE VALUE***",        "dbpassword": "***REMOVED SENSITIVE VALUE***",        "logtimezone": "UTC",        "installed": true,        "mail_smtpmode": "smtp",        "mail_sendmailmode": "smtp",        "mail_from_address": "***REMOVED SENSITIVE VALUE***",        "mail_domain": "***REMOVED SENSITIVE VALUE***",        "default_phone_region": "DE",        "memcache.local": "\\OC\\Memcache\\APCu",        "maintenance": false,        "updater.release.channel": "stable",        "integrity.check.disabled": true,        "theme": "",        "loglevel": 0,        "mail_smtpsecure": "tls",        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",        "mail_smtpport": "25",        "app_install_overwrite": [            "documentserver_community",            "cookbook",            "onlyoffice",            "richdocumentscode",            "occweb"        ],        "encryption.key_storage_migrated": false,        "updater.secret": "***REMOVED SENSITIVE VALUE***"    }}
```

### List of activated Apps

```shell
Enabled:  - accessibility: 1.10.0  - activity: 2.16.0  - bookmarks: 11.0.3  - calendar: 3.5.0  - cloud_federation_api: 1.7.0  - contacts: 4.2.1  - cookbook: 0.9.15  - dav: 1.22.0  - deck: 1.7.1  - encryption: 2.12.0  - event_update_notification: 1.5.0  - federatedfilesharing: 1.14.0  - federation: 1.14.0  - files: 1.19.0  - files_rightclick: 1.3.0  - files_sharing: 1.16.2  - logreader: 2.9.0  - lookup_server_connector: 1.12.0  - notifications: 2.12.1  - oauth2: 1.12.0  - occweb: 0.1.0  - passwords: 2022.9.20  - photos: 1.6.0  - provisioning_api: 1.14.0  - richdocuments: 6.2.0  - richdocumentscode: 22.5.502  - settings: 1.6.0  - tasks: 0.14.4  - theming: 1.15.0  - theming_customcss: 1.12.0  - twofactor_backupcodes: 1.13.0  - updatenotification: 1.14.0  - viewer: 1.8.0  - workflowengine: 2.6.0Disabled:  - admin_audit  - circles: 22.1.1  - comments: 1.11.0  - contactsinteraction: 1.2.0  - dashboard: 7.2.0  - files_external  - files_pdfviewer: 2.3.1  - files_trashbin: 1.8.0  - files_versions: 1.15.0  - files_videoplayer: 1.10.0  - firstrunwizard: 2.7.0  - nextcloud_announcements: 1.7.0  - password_policy: 1.8.0  - privacy: 1.2.0  - recommendations: 0.6.0  - serverinfo: 1.12.0  - sharebymail: 1.8.0  - support: 1.1.0  - survey_client: 1.6.0  - systemtags: 1.11.0  - text: 3.3.1  - user_ldap  - user_status: 1.0.1  - weather_status: 1.0.0
```

### Nextcloud Signing status

```shell
No errors have been found.
```

### Nextcloud Logs

```shell
{"reqId":"YzC2m0xcrePjeKCGcDIMswAAKBE","level":0,"time":"2022-09-25T20:14:19+00:00","remoteAddr":"156.146.33.76","user":"siegfried","app":"encryption","method":"GET","url":"/index.php/settings/admin/logging","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36","version":"24.0.5.1","data":{"app":"encryption"}}
{"reqId":"YzC2m0xcrePjeKCGcDIMswAAKBE","level":0,"time":"2022-09-25T20:14:19+00:00","remoteAddr":"156.146.33.76","user":"siegfried","app":"scss_cacher","method":"GET","url":"/index.php/settings/admin/logging","message":"SCSSCacher::process ordinary check follows","userAgent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36","version":"24.0.5.1","data":{"app":"scss_cacher"}}
{"reqId":"YzC2m0xcrePjeKCGcDIMswAAKBE","level":0,"time":"2022-09-25T20:14:19+00:00","remoteAddr":"156.146.33.76","user":"siegfried","app":"scss_cacher","method":"GET","url":"/index.php/settings/admin/logging","message":"SCSSCacher::process ordinary check follows","userAgent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36","version":"24.0.5.1","data":{"app":"scss_cacher"}}
{"reqId":"YzC2m0xcrePjeKCGcDIMswAAKBE","level":0,"time":"2022-09-25T20:14:19+00:00","remoteAddr":"156.146.33.76","user":"siegfried","app":"logreader","method":"GET","url":"/index.php/settings/admin/logging","message":"Could not find resource file \"/apps/logreader/css/../js/logreader-main.css\"","userAgent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36","version":"24.0.5.1","data":{"app":"logreader"}}
{"reqId":"YzC2m0xcrePjeKCGcDIMswAAKBE","level":0,"time":"2022-09-25T20:14:19+00:00","remoteAddr":"156.146.33.76","user":"siegfried","app":"scss_cacher","method":"GET","url":"/index.php/settings/admin/logging","message":"SCSSCacher::process ordinary check follows","userAgent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36","version":"24.0.5.1","data":{"app":"scss_cacher"}}
{"reqId":"YzC2m0xcrePjeKCGcDIMswAAKBE","level":0,"time":"2022-09-25T20:14:19+00:00","remoteAddr":"156.146.33.76","user":"siegfried","app":"scss_cacher","method":"GET","url":"/index.php/settings/admin/logging","message":"SCSSCacher::process ordinary check follows","userAgent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36","version":"24.0.5.1","data":{"app":"scss_cacher"}}
{"reqId":"YzC2m0xcrePjeKCGcDIMswAAKBE","level":0,"time":"2022-09-25T20:14:19+00:00","remoteAddr":"156.146.33.76","user":"siegfried","app":"scss_cacher","method":"GET","url":"/index.php/settings/admin/logging","message":"SCSSCacher::process ordinary check follows","userAgent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36","version":"24.0.5.1","data":{"app":"scss_cacher"}}
{"reqId":"YzC2m0xcrePjeKCGcDIMtQAAARE","level":0,"time":"2022-09-25T20:14:19+00:00","remoteAddr":"156.146.33.76","user":"siegfried","app":"encryption","method":"GET","url":"/ocs/v2.php/search/providers?from=%2Fsettings%2Fadmin%2Flogging","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36","version":"24.0.5.1","data":{"app":"encryption"}}
{"reqId":"YzC2m0xcrePjeKCGcDIMtgAAAQE","level":0,"time":"2022-09-25T20:14:19+00:00","remoteAddr":"156.146.33.76","user":"siegfried","app":"encryption","method":"GET","url":"/index.php/apps/logreader/settings","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36","version":"24.0.5.1","data":{"app":"encryption"}}
{"reqId":"YzC2m0xcrePjeKCGcDIMtwAAARM","level":0,"time":"2022-09-25T20:14:19+00:00","remoteAddr":"156.146.33.76","user":"siegfried","app":"encryption","method":"GET","url":"/ocs/v2.php/apps/notifications/api/v2/notifications","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36","version":"24.0.5.1","data":{"app":"encryption"}}
{"reqId":"YzC2m0xcrePjeKCGcDIMuAAAARA","level":0,"time":"2022-09-25T20:14:20+00:00","remoteAddr":"156.146.33.76","user":"siegfried","app":"encryption","method":"GET","url":"/index.php/apps/encryption/ajax/getStatus","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36","version":"24.0.5.1","data":{"app":"encryption"}}
{"reqId":"YzC2m0xcrePjeKCGcDIMuQAAAQE","level":0,"time":"2022-09-25T20:14:20+00:00","remoteAddr":"156.146.33.76","user":"siegfried","app":"encryption","method":"GET","url":"/index.php/apps/logreader/get?offset=0&count=50&levels=00111","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36","version":"24.0.5.1","data":{"app":"encryption"}}
{"reqId":"YzC2nUxcrePjeKCGcDIMvgAAAgE","level":0,"time":"2022-09-25T20:14:21+00:00","remoteAddr":"156.146.33.76","user":"siegfried","app":"encryption","method":"GET","url":"/index.php/apps/photos/service-worker.js","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36","version":"24.0.5.1","data":{"app":"encryption"}}
{"reqId":"YzC2pUxcrePjeKCGcDIMxwAAJRA","level":0,"time":"2022-09-25T20:14:29+00:00","remoteAddr":"156.146.33.76","user":"siegfried","app":"encryption","method":"GET","url":"/index.php/apps/cookbook/","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36","version":"24.0.5.1","data":{"app":"encryption"}}
{"reqId":"YzC2pUxcrePjeKCGcDIMxwAAJRA","level":0,"time":"2022-09-25T20:14:29+00:00","remoteAddr":"156.146.33.76","user":"siegfried","app":"no app in context","method":"GET","url":"/index.php/apps/cookbook/","message":"mount at /siegfried/ has no provider set, performing full setup","userAgent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36","version":"24.0.5.1","data":[]}
{"reqId":"YzC2pUxcrePjeKCGcDIMxwAAJRA","level":0,"time":"2022-09-25T20:14:29+00:00","remoteAddr":"156.146.33.76","user":"siegfried","app":"scss_cacher","method":"GET","url":"/index.php/apps/cookbook/","message":"SCSSCacher::process ordinary check follows","userAgent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36","version":"24.0.5.1","data":{"app":"scss_cacher"}}
{"reqId":"YzC2pUxcrePjeKCGcDIMxwAAJRA","level":0,"time":"2022-09-25T20:14:29+00:00","remoteAddr":"156.146.33.76","user":"siegfried","app":"scss_cacher","method":"GET","url":"/index.php/apps/cookbook/","message":"SCSSCacher::process ordinary check follows","userAgent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36","version":"24.0.5.1","data":{"app":"scss_cacher"}}
{"reqId":"YzC2pUxcrePjeKCGcDIMxwAAJRA","level":0,"time":"2022-09-25T20:14:29+00:00","remoteAddr":"156.146.33.76","user":"siegfried","app":"scss_cacher","method":"GET","url":"/index.php/apps/cookbook/","message":"SCSSCacher::process ordinary check follows","userAgent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36","version":"24.0.5.1","data":{"app":"scss_cacher"}}
{"reqId":"YzC2pUxcrePjeKCGcDIMxwAAJRA","level":0,"time":"2022-09-25T20:14:29+00:00","remoteAddr":"156.146.33.76","user":"siegfried","app":"scss_cacher","method":"GET","url":"/index.php/apps/cookbook/","message":"SCSSCacher::process ordinary check follows","userAgent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36","version":"24.0.5.1","data":{"app":"scss_cacher"}}
{"reqId":"YzC2pkxcrePjeKCGcDIMywAAJRA","level":0,"time":"2022-09-25T20:14:30+00:00","remoteAddr":"156.146.33.76","user":"siegfried","app":"encryption","method":"GET","url":"/ocs/v2.php/search/providers?from=%2Fapps%2Fcookbook%2F","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36","version":"24.0.5.1","data":{"app":"encryption"}}
{"reqId":"YzC2poEfdx4r7DJIF8iSTgAAIgA","level":0,"time":"2022-09-25T20:14:30+00:00","remoteAddr":"156.146.33.76","user":"siegfried","app":"encryption","method":"PROPFIND","url":"/remote.php/dav/files/siegfried/","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (Macintosh) mirall/3.6.0git (build 11568) (Nextcloud, osx-21.6.0 ClientArchitecture: x86_64 OsArchitecture: x86_64)","version":"24.0.5.1","data":{"app":"encryption"}}
{"reqId":"YzC2pkxcrePjeKCGcDIMzgAAKRA","level":0,"time":"2022-09-25T20:14:30+00:00","remoteAddr":"156.146.33.76","user":"siegfried","app":"encryption","method":"GET","url":"/index.php/apps/cookbook/webapp/config","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36","version":"24.0.5.1","data":{"app":"encryption"}}
{"reqId":"YzC2pkxcrePjeKCGcDIMzwAAKQE","level":0,"time":"2022-09-25T20:14:30+00:00","remoteAddr":"156.146.33.76","user":"siegfried","app":"encryption","method":"GET","url":"/index.php/apps/cookbook/webapp/categories","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36","version":"24.0.5.1","data":{"app":"encryption"}}
{"reqId":"YzC2pkxcrePjeKCGcDIM0AAAKRM","level":0,"time":"2022-09-25T20:14:30+00:00","remoteAddr":"156.146.33.76","user":"siegfried","app":"encryption","method":"GET","url":"/index.php/apps/cookbook/webapp/recipes","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36","version":"24.0.5.1","data":{"app":"encryption"}}
{"reqId":"YzC2pkxcrePjeKCGcDIM1QAAABA","level":0,"time":"2022-09-25T20:14:30+00:00","remoteAddr":"156.146.33.76","user":"--","app":"encryption","method":"GET","url":"/index.php/apps/theming/manifest/cookbook?v=15","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36","version":"24.0.5.1","data":{"app":"encryption"}}
{"reqId":"YzC2pkxcrePjeKCGcDIM0QAAKRE","level":0,"time":"2022-09-25T20:14:30+00:00","remoteAddr":"156.146.33.76","user":"siegfried","app":"encryption","method":"GET","url":"/ocs/v2.php/apps/notifications/api/v2/notifications","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36","version":"24.0.5.1","data":{"app":"encryption"}}
{"reqId":"YzC2pkxcrePjeKCGcDIM0gAAKQM","level":0,"time":"2022-09-25T20:14:30+00:00","remoteAddr":"156.146.33.76","user":"siegfried","app":"encryption","method":"GET","url":"/index.php/apps/encryption/ajax/getStatus","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36","version":"24.0.5.1","data":{"app":"encryption"}}
{"reqId":"YzC2pkxcrePjeKCGcDIM1gAAKQk","level":0,"time":"2022-09-25T20:14:30+00:00","remoteAddr":"156.146.33.76","user":"siegfried","app":"encryption","method":"POST","url":"/index.php/apps/cookbook/webapp/config","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36","version":"24.0.5.1","data":{"app":"encryption"}}
{"reqId":"YzC2pkxcrePjeKCGcDIM4AAAKQw","level":0,"time":"2022-09-25T20:14:31+00:00","remoteAddr":"156.146.33.76","user":"siegfried","app":"encryption","method":"GET","url":"/index.php/apps/cookbook/webapp/recipes/85664/image?size=thumb16","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36","version":"24.0.5.1","data":{"app":"encryption"}}
{"reqId":"YzC2pkxcrePjeKCGcDIM2AAAKRI","level":0,"time":"2022-09-25T20:14:31+00:00","remoteAddr":"156.146.33.76","user":"siegfried","app":"encryption","method":"GET","url":"/index.php/apps/cookbook/webapp/recipes/83966/image?size=thumb16","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36","version":"24.0.5.1","data":{"app":"encryption"}}
{"reqId":"YzC2pkxcrePjeKCGcDIM2gAAKQE","level":0,"time":"2022-09-25T20:14:31+00:00","remoteAddr":"156.146.33.76","user":"siegfried","app":"encryption","method":"GET","url":"/index.php/apps/cookbook/webapp/recipes/84004/image?size=thumb16","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36","version":"24.0.5.1","data":{"app":"encryption"}}
{"reqId":"YzC2pkxcrePjeKCGcDIM2QAAKRM","level":0,"time":"2022-09-25T20:14:31+00:00","remoteAddr":"156.146.33.76","user":"siegfried","app":"encryption","method":"GET","url":"/index.php/apps/cookbook/webapp/recipes/86022/image?size=thumb16","message":"/appinfo/app.php is deprecated, use \\OCP\\AppFramework\\Bootstrap\\IBootstrap on the application class instead.","userAgent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36","version":"24.0.5.1","data":{"app":"encryption"}}
```

**Comment by Jolopu** [comment]
I upgraded to NC25. The issue is still there as it was with NC 24 and NC 23. No issue with NC22 though.

**Comment by hul51u** [comment]
I upgraded from NC24.0.6 to NC24.0.7. The issue still has been affecting my instance.

**Comment by vasyugan** [comment]
Could this be a duplicate of #33224 ?

**Comment by szaimen** [comment]
Should be resolved with 25.0.4 and 24.0.10

**Comment by faustino-schmutz** [comment]
I'm having the same issue right, No matter how much I increase it it never is enough.
![2023-06-06_12-51](https://github.com/nextcloud/server/assets/130873480/8387ac4e-2500-4272-897b-bfb14dd1dda4)

'Es importante para la seguridad y rendimiento de tu instancia que todo esté correctamente configurado. Para ayudarte con ello estamos haciendo algunas comprobaciones automáticas. Para más información, por favor comprueba la documentación enlazada.
Hay varios avisos relativos a su configuración.

    El módulo PHP OPcache no está configurado adecuadamente. Echa un vistazo a la [documentación↗](https://docs.nextcloud.com/server/26/go.php?to=admin-php-opcache) para más información.
        El buffer de OPcache está casi lleno. Para asegurar que todos los scripts puedan ser retenidos en la caché, se recomienda aplicar opcache.memory_consumption a su configuración de PHP con un valor superior a 32.

Por favor, vuelve a consultar las [guías de instalación ↗](https://docs.nextcloud.com/server/26/go.php?to=admin-install), y comprueba posibles errores y advertencias en el [registro](https://nextcloud.globspot.tech/index.php/settings/admin/logging).

Compruebe la seguridad de su Nextcloud mediante [nuestro escaneo de seguridad ↗](https://scan.nextcloud.com/).'
